.github/workflows/ci.yml

```diff
@@ -20,9 +23,14 @@ jobs:
         uses: actions/setup-node@v4
         with:
           node-version-file: ".nvmrc"
+          cache: "npm"
 
       - name: Install dependencies
-        run: npm install
+        run: npm ci
```

Comment:
why is this changed?

Reply:
https://letmegooglethat.com/?q=npm+install+vs+npm+ci